### ISSUE TITLE
Unable to parse `traceroute -a` when a hop has multiple ASNs

### ISSUE THREAD

**Issue by rarylson** [lbenitez000/trparse#9]
Example that works (although ASN should belong to a probe, not to a hop):

```
 9  [AS19905] xe-0-1-2.2028.jr01vr-napbr-fly.fly.com.br (177.93.238.14)  21.291 ms  21.860 ms  21.067 ms
10  [AS22356] xe-0-0-0.96.jr03-napbr-db.durand.com.br (200.170.83.97)  15.772 ms  736.220 ms  23.648 ms
```

Example that do not work:

```
11  [AS19905] ae0.900.jr02-tvt-spo.durand.com.br (177.190.108.50)  26.945 ms
    [AS3356] 4.16.168.34 (4.16.168.34)  192.924 ms  187.665 ms
12  [AS16509] 52.95.54.156 (52.95.54.156)  181.465 ms
    [AS16509] 52.95.55.2 (52.95.55.2)  185.864 ms  184.844 ms
```

To generate that output, I run the command `traceroute -a vialink.com.br` in MACOS X.

**Comment by rarylson** [comment]
Although they are different bugs, this problem can be solved together with #5. Both are parse bugs.

**Comment by rarylson** [comment]
Actually, this is a duplicate of #7. However, here, I'm putting a more clear example.

**Comment by rarylson** [comment]
As an additional note, the ASN zero ('[AS0]') is not printed (`__str__` hop/probe methods) because `0` and `None` both evaluate to boolean false. This is also fixes in 32cae20.

**Comment by rarylson** [comment]
It seems to be fixed now (test case `07.txt`), in v0.3.0. Closing the issue.
